### PR TITLE
[NotationV0] bump: bump up notation version to v1.1.0

### PR DESCRIPTION
**Pull request by JeyJeyGao**
**Task name**: NotationV0

**Description**: 
- update notation version v1.1.0
- added a script to generate version file
- updated README to explain how to bump up notation version
- removed user document from README file

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) No need.

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by DergachevE — APPROVED** [review]

File: Tasks/NotationV0/scripts/generate_checksum.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import sys
+import json
+
+def build_url(name, version, filename):
+    return {
+        "notation": lambda : f'https://github.com/notaryproject/notation/releases/download/v{version}/{filename}',
+        "notation-azure-kv": lambda :f'https://github.com/Azure/notation-azure-kv/releases/download/v{version}/{filename}'
+    }[name]()
+
+
```
Comment (shizhMSFT) [inline Tasks/NotationV0/scripts/generate_checksum.py:27]: nit: remove extra lines

**Review by shizhMSFT — APPROVED** [review]
LGTM but IANAM

File: Tasks/NotationV0/scripts/generate_checksum.py (hunk 1 above)
Comment (JeyJeyGao) [reply Tasks/NotationV0/scripts/generate_checksum.py]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM but IANAM

**Pull request merged**